cs251tk/toolkit/args.py

```diff
@@ -18,7 +19,12 @@ def get_args():
     parser = argparse.ArgumentParser(description='The core of the CS251 toolkit')
     parser.add_argument('input', nargs='*',
                         help='A mixed list of students and assignments')
-    parser.add_argument('--debug', action='store_true', help='enable debugging mode (throw errors, implies -w1)')
+    parser.add_argument('--debug', action='store_true',
+                        help='enable debugging mode (throw errors, implies -w1)')
+
+    specs = argparse.ArgumentParser(description='control the homework specs')
+    specs.add_argument('--course', default='sd', choices=['sd', 'hd'],
+                       help='Which course to evaulate (this sets a default stogit url)')
```

Comment:
Very good.  One possibility to consider is merging all course specs together in one repository to help unify everything.  Then, this could be more dynamic and make this more general-purpose.